src/Service/Generate/PhpRenderer.php

```diff
@@ -37,4 +37,11 @@ public function render(string $file, array $params = []): string
             throw $e;
         }
     }
+
+    private function renderer(string $file, array $params): void
```

Comment:
Use `func_get_arg()` instead of explicitly specifying parameters. This prevent pass parameters `$file` and `$params` to file.